packages/camera/camera/example/pubspec.yaml

```diff
@@ -27,6 +27,7 @@ dev_dependencies:
     sdk: flutter
   integration_test:
     sdk: flutter
+  leak_tracker_flutter_testing: any
```

Comment:
Why use `any` rather than `^3.0.9`?

Reply:
This is what is recommended in the documentation (https://github.com/dart-lang/leak_tracker/blob/main/doc%2Fleak_tracking%2FDETECT.md) because the version is defined by the Flutter SDK.